Week1/credit/credit.c

```diff
@@ -0,0 +1,85 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    // input creditcard number
```

Comment:
This comment for example just states the obvious: "The next line gets the input for the creditcardnumber"